packages/@n8n/task-runner-python/src/main.py

```diff
@@ -3,35 +3,54 @@
 import os
 import sys
 
-from .constants import ENV_TASK_BROKER_URI, ENV_GRANT_TOKEN, DEFAULT_TASK_BROKER_URI
-from .task_runner import TaskRunner
-
-logging.basicConfig(
-    level=logging.INFO, format="%(asctime)s - %(name)s - %(levelname)s - %(message)s"
+os.environ["WEBSOCKETS_MAX_LOG_SIZE"] = "256"
```

Comment:
The [default](https://websockets.readthedocs.io/en/stable/topics/logging.html) of `75` truncates too much.